zerver/migrations/0310_userprofile_enable_notification_on_unsubscribe_stream.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.17 on 2020-12-28 14:57
```

Comment:
Do we need 2 migrations here?

Reply:
Actually initially there was one migration but later some conflicts arised in migration so I had to run <code>makemigrations --merge</code> which generated the extra migration file.